### PR TITLE
feat: add `en-GB` translations

### PR DESCRIPTION
**Pull request by digital-zen**
# Description of change
Please write a summary of your changes and why you made them.

## Links to any relevant issues
Be sure to reference any related issues by adding `fixes issue #`.

## How the change has been tested
Describe the tests that you ran to verify your changes.
Make sure to provide instructions for the maintainer as well as any relevant configurations.

## Definition of Done checklist
Add an `x` to the boxes that are relevant to your changes.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: unime/src/i18n/en-GB/index.ts [hunk 1]
```diff
@@ -4,12 +4,27 @@ import en from '../en';
...
+  ONBOARDING: {
+    CUSTOMIZE: {
+      NAVBAR_TITLE: 'Customisation',
+    },
+    PLEDGE: {
+        DESCRIPTION:
+          'Your data belongs to you and only you decide who you share it with. Full stop. In fact, your data never even touches any of our systems - unless you opt-in to one of the cloud storage options.',
+      },
+    ITEM_2: {
+        TITLE: 'We will not add trackers',
```
Comment (daniel-mader) [inline unime/src/i18n/en-GB/index.ts]: You can remove this `TITLE` entry if there isn't any difference to the base English.

File: unime/src/i18n/en-GB/index.ts [hunk 2]
```diff
@@ -4,12 +4,27 @@ import en from '../en';
...
-  //   CUSTOMIZE: {
-  //     NAVBAR_TITLE: 'Customisation',
-  //   },
-  // },
+  ...(en as Translation),
+  ONBOARDING: {
+    CUSTOMIZE: {
+      NAVBAR_TITLE: 'Customisation',
+    },
+    PLEDGE: {
```
Comment (daniel-mader) [inline unime/src/i18n/en-GB/index.ts:15]: I think there's something off with the nesting a little, the base structure is as follows:

```
PLEDGE: {
  ITEM_1: {
    DESCRIPTION: '...'
  },
  ITEM_2: {
    DESCRIPTION: '...'
  }
}
```

Your current changes would introduce:

```
PLEDGE: {
  DESCRIPTION: '...'
}
```

Solution: add a wrapping `ITEM_1` around the first `DESCRIPTION`.

**Review by daniel-mader — CHANGES_REQUESTED** [review]
Thanks for the contribution! 🥳

**Comment by digital-zen** [comment]
Best day ever

**Review by daniel-mader — APPROVED** [review]

**Review by JelleMillenaar — APPROVED** [review]

**Pull request merged**